tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java

```diff
@@ -360,7 +360,7 @@ public void printWindow() {
             long ellapsed = System.currentTimeMillis() - windowStart;
             double recsPerSec = 1000.0 * windowCount / (double) ellapsed;
             double mbPerSec = 1000.0 * this.windowBytes / (double) ellapsed / (1024.0 * 1024.0);
-            System.out.printf("%d records sent, %.1f records/sec (%.2f MB/sec), %.1f ms avg latency, %.1f max latency.%n",
+            System.out.printf("%d records sent, %.1f records/sec (%.2f MiB/sec), %.1f ms avg latency, %.1f max latency.%n",
```

Comment:
I wonder how many people are familiar with `MiB`. Maybe we should just change the computation so that `MB` is accurate?

Reply:
I'm not going to defend the choices the standards bodies made.  They took an ambiguous situation and muddied it further. 
 * `MiB` unambiguously means `1024^2`, but is not adopted everywhere.
 * `MB` *probably* means `1024^2` in most contexts that are not selling hard drives, but technically means `1000^2`.
 * There is no unambiguous way to express `1000^2` with a prefix.
 * Old school cli utilities sometimes use a single capital letter `M` to mean `1024^2`, but that isn't well supported by standards and is awkward with a rate.

The `Mi` style prefixes were standardized in the late 1990s.  Ubuntu and most GNU utilities adopted them circa 2010. At this point no amount of righteousness will sway the standards bodies to revisit all of this, even if the conspiratorial minded think it was all a plot by thieving hard drive manufactures.

I think using `MB` to mean `1000^2` while technically correct is probably a more confusing change since it isn't what most people expect. Perf test results would silently change from one version of Kafka to the next.  I don't think `MB` means `1000^2` anywhere else in Kafka.  It's probably convenient for humans that that perf test always reports the same scale of units, but an alternative change would be to have `org.apache.kafka.common.utils.Utils` always use the `Mi` style binary prefix units and then use that method here.